bindings/ruby/extconf.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'mkmf'
+
+$LDFLAGS << ' -lc++'
```

Comment:
Good question, I'll check with clang on OS X tomorrow and report back.

Part of the concern was not what it is now, it was lack of visibility. Between compilers, between targets, between versions, the set of flags determined by the driver will change, but users aren't really aware of those flags, and how they change. Is it better to depend on the compiler always knowing best, or better to try to leave the compiler out and pass the one library to the linker?